src/libs/actions/IOU.js

```diff
@@ -221,11 +221,15 @@ function buildOnyxDataForMoneyRequest(
                           [chatCreatedAction.reportActionID]: {
                               errors: ErrorUtils.getMicroSecondOnyxError('iou.error.genericCreateFailureMessage'),
                           },
+                          [reportPreviewAction.reportActionID]: {
+                              errors: ErrorUtils.getMicroSecondOnyxError(null),
+                          },
                       }
-                    : {}),
-                [reportPreviewAction.reportActionID]: {
-                    created: reportPreviewAction.created,
-                },
+                    : {
+                          [reportPreviewAction.reportActionID]: {
+                              created: reportPreviewAction.created,
+                          },
+                      }),
```

Comment:
This is done so we can set the null error to the report action in the if condition. We need this for the opacity.

Reply:
I don't think we need to reset `created` back to the old value in the if condition since the report action preview will definitely be new if the chat is new.

Reply:
This preview report action is not necessary linked to the chat. We have another parameter `isNewReportPreviewAction`. I think we should reset this report action as follow:
`isNewReportPreviewAction ? null : {created: reportPreviewAction.created}`

Reply:
That parameter is for if the chat is not new and the report preview is newly created, for example if a split is created with existing members. If the chat is new then then the report preview is definitely new in this case.

We can't reset the report preview action to null as it's still required to render.

Reply:
or actually set an error instead of `null`

Reply:
So I'll reset created inside the if condition?

```js
{
    onyxMethod: Onyx.METHOD.MERGE,
    key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReport.reportID}`,
    value: {
        ...(isNewChatReport
            ? {
                    [chatCreatedAction.reportActionID]: {
                        errors: ErrorUtils.getMicroSecondOnyxError('iou.error.genericCreateFailureMessage'),
                    },
                    [reportPreviewAction.reportActionID]: {
                        created: reportPreviewAction.created,  // Added this
                        errors: ErrorUtils.getMicroSecondOnyxError(null),
                    },
                }
            : {
                    [reportPreviewAction.reportActionID]: {
                        created: reportPreviewAction.created,
                    },
              }),
    },
}
```

This would keep `created` the same as before.

Reply:
Oh I see, nvm, do nothing. There is no point in resting the `created` to the current `created` field (same value)